bin/genesis

```diff
@@ -799,13 +799,24 @@ create_normal_environment() {
 EOF
 	done
 
-	local director_uuid
-	director_uuid=$(bosh status --uuid)
-	if [[ "$?" -eq 0 ]] ; then
-		cat <<EOF > ${root}/director.yml
+	# As a convenience, try to get the BOSH Director UUID, but only if the director in set and reachable.
+	if [[ -f $HOME/.bosh_config ]] ; then
+		local director_url
+		local director_address
+		local fetched_director_id
+
+		director_url=$(spruce json ~/.bosh_config | jq -r ".target")
+		director_address=$(echo $director_url | sed -ne 's/^https\{0,1\}:\/\/\(.*\):[0-9]\{1,\}$/\1/p')
+		if [[ -n "$director_address" ]] &&\
+			 ping $director_address -c1 -W1 >/dev/null 2>&1 &&\
+			 curl -k -m5 ${director_url}/info >/dev/null 2>&1
+		then
+			fetched_director_uuid=$(bosh status --uuid) && director_uuid=${fetched_director_uuid}
```

Comment:
Yeah, that's a remnant of the previous way I was doing things.  New way is cleaner, but left that in.  It works as-is, but I will remove it.